packages/@adobe/spectrum-css-temp/components/button/spectrum-button.css

```diff
@@ -0,0 +1,375 @@
+:root {
+  /* RSP stopgap since we don't have spectrum-button css vars */
+  --spectrum-button-cta-icon-color: var(--spectrum-global-color-static-white);
+  --spectrum-button-cta-background-color: var(--spectrum-semantic-cta-color-background-default);
+  --spectrum-button-cta-border-color: var(--spectrum-semantic-cta-color-background-default);
+  --spectrum-button-cta-text-color: var(--spectrum-global-color-static-white);
+  --spectrum-button-cta-icon-color-hover: var(--spectrum-global-color-static-white);
+  --spectrum-button-cta-background-color-hover: var(--spectrum-semantic-cta-color-background-hover);
```

Comment:
Created a new file to store the spectrum-button vars and added it to the `vars.css` import so I wouldn't have to paste it into skin/index. Lemme know if there is a better way to go about this.

Note that these vars are sourced from spectrum-css master.

Reply:
This is repeated for the other components as well